File: redwood-yoga-java/src/main/java/io/github/orioncraftmc/meditate/internal/enums/YGDimension.java
package io.github.orioncraftmc.meditate.internal.enums;

public enum YGDimension {
    YGDimensionWidth,
    YGDimensionHeight;

    public static final int SIZE = java.lang.Integer.SIZE;

    public static YGDimension forValue(int value) {
        return values()[value];
    }

    public int getValue() {
        return this.ordinal();
    }
}

File: redwood-yoga/src/commonMain/kotlin/app/cash/redwood/yoga/internal/enums/YGDimension.kt
package app.cash.redwood.yoga.internal.enums

enum class YGDimension {
    YGDimensionWidth, YGDimensionHeight;

    fun getValue(): Int {
        return ordinal
    }

    companion object {
        fun forValue(value: Int): YGDimension {
            return values()[value]
        }
    }
}
